aea/configurations/schemas/definitions.json

```diff
@@ -218,6 +219,9 @@
         "not_after": {
           "type": "string"
         },
+        "message_format": {
+          "type": "string"
```

Comment:
upper bound on length?

Reply:
Arbitrary? The message format is implicitly enforced by the system it is used on.